Sources/AppStoreConnectCLI/Services/Operations/ListBuildsOperation.swift

```diff
@@ -0,0 +1,88 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+
+struct ListBuildsOperation: APIOperation {
+
+  struct Options {
+    let appId: [String]
+    let expired: [String]
+    let preReleaseVersion: [String]
+    let buildNumber: [String]
+    let betaReviewState: [String]
+    let limit: Int?
+  }
+
+  enum ListBuildsError: LocalizedError {
+    case noBuildExist
+
+    var errorDescription: String? {
+      switch self {
+      case .noBuildExist:
+        return "No build exists"
+      }
+    }
+  }
+
+  private let options: Options
+
+  init(options: Options) {
+    self.options = options
+  }
+
+  func execute(with requestor: EndpointRequestor) throws -> AnyPublisher<[Build], Error> {
+
+    var filters = [ListBuilds.Filter]()
+
+    if !options.appId.isEmpty {
+      filters += [ListBuilds.Filter.app(options.appId)]
```

Comment:
I have been reducing this to one line by using a ternary:
```swift
filters += options.filterAppIds.isEmpty ? [] : [ListBuilds.Filter.app(options.filterAppIds)]
```
Minimally less efficient but it does save some space